packages/node/test/integration/utils.ts

```diff
@@ -190,6 +201,24 @@ export function makeDepositRequest(
   });
 }
 
+export function makeWithdrawCommitmentRequest(
```

Comment:
`request` is not defined as part of the API, this should be called `makeWithdrawCommitmentRpc`

Reply:
Agree this could be more accurately named, but `makeWithdrawCommitmentRpc` would also be inaccurate because the call itself isn't being made by this function.

If anything, this should be named `constructWithdrawCommitmentRpc` because the function is constructing the RPC to be made.

Reply:
I'll do the renaming in a separate PR to reduce the noise here.

Reply:
https://github.com/counterfactual/monorepo/pull/2175